### PR TITLE
Health monitor and rosmon bug fixes

### PR DESCRIPTION
**Pull request by patapufate**
After testing health monitor these bugs appeared:

- rosmon topics is rosmon/state instead rosmon/Sate
- Health monitor now publish to diagnostics using code:description
- bug fix with diagnostics_tools in code value check.

### CODE REVIEW

File: health_monitor/src/health_monitor.cpp [hunk 1]
```diff
@@ -79,7 +79,8 @@ HealthMonitor::HealthMonitor(ros::NodeHandle &nodeHandle) : nodeHandle(nodeHandl
     diagnosticsUpdater.add(publisher_reportFault.add_check<diagnostic_tools::PeriodicMessageStatus>(
         "rate check", diagnostic_tools::PeriodicMessageStatusParams{}
                           .min_acceptable_period(minReportFaultRate)
-                          .max_acceptable_period(maxReportFaultRate)));
+                          .max_acceptable_period(maxReportFaultRate)
+                          .abnormal_diagnostic({diagnostic_tools::Diagnostic::ERROR, 1})));
```
Comment (hidmic) [inline health_monitor/src/health_monitor.cpp:83]: @patapufate is this change necessary?

**Review by hidmic — COMMENTED** [review]
@patapufate mind to repharse the PR title for clarity?

File: health_monitor/src/health_monitor.cpp (hunk 1 above)
Comment (patapufate) [reply health_monitor/src/health_monitor.cpp:83]: if we are gonna use diagnostics_tools with error:description msg in a future for diagnose health_monitor package, yes (from example, from mission_manager). if not, we could also remove all diagnostics from publisher_reportFault

Comment (hidmic) [reply health_monitor/src/health_monitor.cpp:83]: But do we have a fault defined for the `health_monitor`? We can generate a diagnostic w/o necessarily having a user facing fault for it.

Comment (patapufate) [reply health_monitor/src/health_monitor.cpp:83]: no. we don't have any fault definition for health monitor. I'll remove.

**Review by hidmic — APPROVED** [review]
LGTM !
